run_test.sh

```diff
@@ -14,15 +14,5 @@ export DBT_PROFILES_DIR=.
 # Show the location of the profiles directory and test the connection
 dbt debug --target $1
 
-_models=""
-_seeds="--full-refresh"
-if [[ ! -z $2 ]]; then _models="--models $2"; fi
-if [[ ! -z $3 ]]; then _seeds="--select $3 --full-refresh"; fi
-
 dbt deps --target $1 || exit 1
-dbt seed --target $1 $_seeds || exit 1
-if [ $1 == 'redshift' ]; then
-    dbt run -x -m test_insert_by_period --full-refresh --target redshift || exit 1
-fi
-dbt run -x --target $1 $_models || exit 1
-dbt test -x --target $1 $_models || exit 1
+dbt build --target $1 --full-refresh || exit 1
```

Comment:
Suggestion below to add newline to remove the red circle icon at the bottom of the file:

<img width="300" alt="image" src="https://user-images.githubusercontent.com/44704949/235127215-6b11f62a-2137-4026-96a1-bee055181d36.png">

```suggestion
dbt build --target $1 --full-refresh || exit 1

```